src/main/java/it/gov/pagopa/payhub/auth/service/a2a/ClientServiceImpl.java

```diff
@@ -49,4 +50,11 @@ public Optional<Client> getClientByClientId(String clientId) {
 		return clientRetrieverService.getClientByClientId(clientId);
 	}
 
+	public ClientDTO verifyCredentials(String clientId, String clientSecret) {
+		return this.getClientByClientId(clientId)
```

Comment:
the `this` keyword is superfluous
```suggestion
		return getClientByClientId(clientId)
```